telegram/_dice.py

```diff
@@ -93,3 +105,17 @@ def __init__(self, value: int, emoji: str, *, api_kwargs: JSONDict = None):
     """
     ALL_EMOJI: ClassVar[List[str]] = list(constants.DiceEmoji)
     """List[:obj:`str`]: A list of all available dice emoji."""
+
+    MIN_VALUE: ClassVar[int] = constants.DiceLimit.MIN_VALUE
+    """:const:`telegram.constants.DiceLimit.MIN_VALUE`"""
+
+    MAX_VALUE_DICE_DARTS_BOWLING: ClassVar[int] = constants.DiceLimit.MAX_VALUE_DICE_DARTS_BOWLING
```

Comment:
I'd vote to split that up into `max_value_{dice, darts, bowling}`, even though the value is repeated. If a new emoji is to be added in the future with the same max value, we'd have to rename the constant otherwise …
same for baseketball_football and also in the `constants` file

Reply:
> However, this will lead to docstring of Dice

I'd be fine with just linking one constant, e.g. `1-Dice.MAX_VALUE for dice, darts and bowling`. The link will take the people to the place where the other constants are defined, so they'll see what's going on.

> because of line breaks in docstrings I now get an extra whitespace rendered between min and max value

Maybe a backslash `\` at the end of the line? the whitespace wouldn't be a deal breaker for me though …

Reply:
> Maybe a backslash \ at the end of the line? the whitespace wouldn't be a deal breaker for me though …

It's strange but it doesn't work - maybe because of references. I know it does work
in URLs, for example. But yes, never mind.

Reply:
Done